packages/next/src/experimental/testmode/playwright/index.ts

```diff
@@ -39,17 +39,14 @@ export const test = base.test.extend<
     { scope: 'worker', auto: true },
   ],
 
-  next: [
-    async ({ nextOptions, _nextWorker, page }, use, testInfo) => {
-      await applyNextFixture(use, {
-        testInfo,
-        nextWorker: _nextWorker,
-        page,
-        nextOptions,
-      })
-    },
-    { auto: true },
```

Comment:
removing this because it forces all `test(...)` invocations to use the test proxy instead of just the `test(...)` invocations that use this fixture